### ISSUE TITLE
The default prompt should show if there are any stashed files

### ISSUE THREAD

**Issue by solazio** [warpdotdev/Warp#2388]
### Describe the solution you'd like?

```text
I would like to see the number (if there are any) of stashed files in the prompt.
```

### Is your feature request related to a problem? Please describe.

```text
You have no indication if there are any stashed files.
```

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Thanks for this feature request! Please :+1:  and subscribe to this request as it helps us gauge interest. 

We'll post any updates on this thread.

**Comment by leontedev** [comment]
This is the only thing holding me back from switching from iTerm2 right now. In iTerm I have `*1` shown at the end of the prompt to know how many stashed items there are.

<img width="232" alt="image" src="https://user-images.githubusercontent.com/20252802/230885685-5df03468-4666-4968-a06e-68adb7a765f8.png">

**Comment by sergiivelykodnyi** [comment]
Yes, it would be nice to have. I was very happy to see that the Warp prompt got this new Git Branch Status block with ahead/behind commits. I'm looking forward to when you add a stash block as well.
